.github/workflows/phpcs.yml

```diff
@@ -48,10 +51,21 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-composer-
 
+      - name: Install 1Password CLI
+        if: steps.changed-files.outputs.phpcs == 'true'
+        uses: 1password/install-cli-action@v1
+
+      - name: Create auth.json from 1Password
+        if: steps.changed-files.outputs.phpcs == 'true'
+        env:
+          OP_SERVICE_ACCOUNT_TOKEN: ${{ secrets.OP_SERVICE_ACCOUNT_TOKEN }}
+        run: |
+          op user get --me
```

Comment:
Q: Do we want to output the service account user for each run? It's somewhat helpful, but also noisy. Thoughts?